feature_importance.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+# coding: utf-8
```

Comment:
You don't need those and the path may not always be /usr/bin/env python for python2 and 3. what you wrote is only for Python2.